frontend/src/Student/components/ShowCard.js

```diff
@@ -136,8 +136,7 @@ const SubmittedEntries = ({ show }) =>
         <EntryThumb entry={entry} />
         {/* If after entry end and before judging end (or if the show is not finalized),
           display "Pending", else display invited or not invited */
-          moment().isBetween(show.entryEnd, show.judgingEnd) ||
-        !show.finalized ? (
+          moment().isBetween(show.entryEnd, show.judgingEnd) || (!show.finalized && moment().isAfter(show.entryEnd)) ? (
```

Comment:
I think this is more readable as:

```javascript
moment().isAfter(show.entryEnd) && (moment().isBefore(show.judgingEnd) || !show.finalized)
```

I think they're equivalent.

Reply:
I'm a bit confused why this was changed in this PR. What is the change attempting to do?

From my understanding, we should show "Pending" after a student makes a submission. Then, in order for invitations to be visible, judging must be over and the admin must have finalized invitations.

Reply:
Consider moving this into a helper function instead of nested ternaries, as it's becoming confusing to understand at a glance.

Reply:
Yeah, I just saw it as an issue when I started up. It's a one line fix, so I changed it, thinking, "clean up as you go". Now that I look back though, it's totally a not realistic case (show finalized before entry end), so I'll remove it.